genesyscloud/telephony_providers_edges_edge_group/genesyscloud_telephony_providers_edges_edge_group_init.go

```diff
@@ -0,0 +1,14 @@
+package telephony_providers_edges_edge_group
+
+import (
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+func SetRegistrar(l registrar.Registrar) {
+	l.RegisterDataSource("genesyscloud_telephony_providers_edges_edge_group", DataSourceEdgeGroup())
+
+	l.RegisterResource("genesyscloud_telephony_providers_edges_edge_group", ResourceEdgeGroup())
+
+	l.RegisterExporter("genesyscloud_telephony_providers_edges_edge_group", EdgeGroupExporter())
+
```

Comment:
you dont need this whitespace

Reply:
Actually scratch that, what is this file? `SetRegistrar` is generally in the schema file